map_model/src/connectivity/mod.rs

```diff
@@ -77,24 +77,67 @@ pub fn all_vehicle_costs_from(
         }
     }
 
-    if let Some(start_road) = bldg_to_road.get(&start) {
-        let graph = build_graph_for_vehicles(map, constraints);
-        let cost_per_road = petgraph::algo::dijkstra(&graph, *start_road, None, |(_, _, mvmnt)| {
```

Comment:
The petgraph crate doesn't allow multiple starting nodes for Dijkstra, or any other similar floodfill algorithm. I switched to implementing Dijkstra's algorithm directly for more control. I had already done this for the walking implementation earlier, because I wanted early termination of the search when the cost exceeds 15m. It'd be nice to not have these two duplicate implementations of Dijkstra's floating around, but they're simple, and I don't want to reinvent some of the genericness that petgraph has.